ext/standard/tests/streams/gh15937.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+GH-15937 (stream overflow on timeout setting)
+--FILE--
```

Comment:
```suggestion
--SKIPIF--
<?php if (getenv("SKIP_ONLINE_TESTS")) die("skip online test");?>
--FILE--
```
Alternatively, don't connect to example.com, but to some test server (possibly using sapi/cli/tests/php_cli_server.inc).